compiler/src/dotty/tools/dotc/transform/ArrayApply.scala

```diff
@@ -0,0 +1,69 @@
+package dotty.tools.dotc
+package transform
+
+import core._
+import MegaPhase._
+import Contexts.Context
+import Symbols._
+import Types._
+import StdNames._
+import ast.Trees._
+import dotty.tools.dotc.ast.tpd
+
+import scala.reflect.ClassTag
+
+
+/** This phase rewrites calls to `Array.apply` to primitive array instantion.
```

Comment:
Typo: instantiation. Also "primitive array" usually means array of primitive so I'd use another term.

Reply:
Updated